contents/docs/libraries/react-native/index.mdx

```diff
@@ -15,22 +15,104 @@ features:
+| `captureNativeAppLifecycleEvents`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                  | Captures native app lifecycle events such as Application Installed, Application Updated, Application Opened, Application Became Active, and Application Backgrounded. By default, this is false. If you're already using the 'captureLifecycleEvents' options with 'initReactNativeNavigation' or 'PostHogProvider', you should not set this to true, otherwise you may see duplicated events.                   |
+| `disableGeoip`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                                     | When true, disables automatic GeoIP resolution for events and feature flags.                                                                                                                                                                                                                              |
 ## Capturing events
 
 import RNSendEvents from '../../integrate/send-events/_snippets/send-events-react-native.mdx'
 
 <RNSendEvents />
 
+## Autocapture 
+
+PostHog autocapture automatically tracks the following events for you:
+
+-   **Application Opened** - when the app is opened from a closed state
+-   **Application Became Active** - when the app comes to the foreground (e.g. from the app switcher)
+-   **Application Backgrounded** - when the app is sent to the background by the user
+-   **Application Installed** - when the app is installed.
+-   **Application Updated** - when the app is updated.
+-   **$screen** - when the user navigates (if using `@react-navigation/native` or `react-native-navigation`)
+-   **$autocapture** - touch events when the user interacts with the screen
+
+With autocapture, all touch events for children of `PosthogProvider` are tracked, capturing a snapshot of the view hierarchy at that point. This enables you to create [insights](/docs/product-analytics/insights) in PostHog without having to add custom events.
+
+PostHog will try to generate a sensible name for the touched element based on the React component `displayName` or `name`. If you prefer, you can set your own name using the `ph-label` prop:
+
+```react-native
+<View ph-label="my-special-label"></View>
+```
+
+### Autocapture configuration
+
+You can tweak how autocapture works by passing custom options.
+
+```react-native
+<PostHogProvider apiKey="<ph_project_api_key>" autocapture={{
+    captureTouches: true, // If you don't want to capture touch events set this to false
+    captureLifecycleEvents: true, // If you don't want to capture the Lifecycle Events (e.g. Application Opened) set this to false
+    captureScreens: true, // If you don't want to capture screen events set this to false
```

Comment:
Do we need the comments for each of these, slightly redundant with the names.

Reply:
The main part is beyond a scroll too

![CleanShot 2024-06-06 at 08 30 28](https://github.com/PostHog/posthog.com/assets/34755028/76491749-3431-4ddf-95ea-c753b8223e79)